src/php/Reader.php

```diff
@@ -129,7 +129,7 @@ public function readExpression(Generator $tokenStream)
                     throw $this->buildReaderException('Unterminated list');
 
                 case Token::T_QUOTE:
-                    return $this->readWrap($tokenStream, 'quote');
+                    return $this->readWrap($tokenStream, Symbol::NAME_QUOTE);
```

Comment:
@jenshaase  I'm not sure about this one 🤔 because what's about ''unquote'? Should we treat it as a symbol as well?

Reply:
Yes, `unquote`, `quote` and `unquote-splicing` are all Symbol names. They are mostly used in the Quasiqoute class.

Reply:
Ok, then I will create those constants as well :)